JenkinsfileRelease

```diff
@@ -0,0 +1,18 @@
+#!groovy
+
+node {
+
+    load "$JENKINS_HOME/jobvars.env"
+
+        stage('Checkout') {
+            checkout scm
+            sh 'git checkout master'
```

Comment:
i suppose that the branch can be specified in jenkins job
just like [here](https://github.com/reportportal/service-api/blob/453abc4a2681dc06db3a011c2a5b7a7288cb0934/Jenkinsfile#L8)

Reply:
done